provider/service_resources.go

```diff
@@ -34,7 +34,24 @@ func resourceService() *schema.Resource {
 			"service_tier": {
 				Type:     schema.TypeInt,
 				Optional: true,
-				Default: 5,
+				Default:  5,
+			},
+			"owner": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
```

Comment:
Is the name attribute necessary here? Is this for referencing by other resources?